README.md

```diff
@@ -5,21 +5,21 @@ How does AirBnB create a separate url for every property it hosts on its site? W
 
 ## Objectives
 
-+ Understand the purpose of dynamic routes.
++ Explain the purpose of dynamic routes.
 + Create dynamic routes in the controller.
 + Show the relationship between dynamic routes and the browser URL.
 
 ## Setup
 
-To code along, fork and clone this lab. Run `bundle install` to make sure all of your dependencies are installed. Run `shotgun` to make sure that your application can run.
+To code along, fork and clone this lab. Run `bundle install` to make sure all of your dependencies are installed. Run `shotgun` to make sure that your application can run. There are tests, so make sure you're running `learn` periodically to make sure you code is behaving as expected.
 
 ### Starter Code
 
-In App.rb, we have two routes:
+Open up `app.rb` in your text editor. You'll notice two routes, `get '/hello'` and `get '/hello/:name'`.
 
-+ `get '/hello'` which returns the string "hello world" in the browser when we go to the url. This is an example of static routing, which we've seen.
+The first route is familiar looking to us. It returns the string "hello world" in the browser when we go to the url. This is an example of static routing, which we've seen.
```

Comment:
Explain a bit more of the reason for why dynamic routes are awesome. Something like "Eventually we are going to need to capture data from the user. We need to know *who* they want to say hello to. There are a few ways to get this information, the easiest is built right into the url. They are already typing that url into the box at the top of their browser, let's use it to get a bit more information"